crates/uv-python/src/discovery.rs

```diff
@@ -944,11 +945,25 @@ pub(crate) fn find_python_installation(
             continue;
         }
 
+        // If it's an alternative implementation, and alternative implementations aren't allowed
+        // skip it; we avoid querying these interpreters at all if they're on the search path and
```

Comment:
Can we include a little more info on what "aren't allowed" means here, and what "opt-in" means for alternative implementations?

Reply:
Note this roughly matches the commentary above about pre-releases, but neither is explained in detail. I'll see what I can do.